docs/microservices/users.md

```diff
@@ -220,3 +222,12 @@ See user model
 | 404 | User not found |
 | 422 | Missing or invalid userId/problemSetId in body |
 | 500 | Internal server error |
+
```

Comment:
Add a quick one sentence explanation of what this route does. Do the same for the other PATCH routes too, as it's not necessarily clear right away

Reply:
we don't do this for any other routes though (users, problems, or problemSets)

Reply:
CRUD routes are self-explanatory imo, but these are different. I added an explanation to the problems/<id>/exists route since it wasn't CRUD. These routes are also not CRUD, so I feel like they need more explanation.

Tbh we maybe should add descriptions to all routes, even the CRUD ones. But I think at the very least we should explain these. I know that even as someone who's been on this project from the start, I was confused by what the other PATCH routes did in the user's microservice.